components/sensor/gdk101.rst

```diff
@@ -0,0 +1,135 @@
+FTLab GDK101 Gamma Radiation Sensor Module
+==========================================
+
+.. seo::
+    :description: Instructions for setting up GDK101 Gamma Radiation Sensor Module
+    :image: gdk101.jpg
+    :keywords: gdk101
+
+The **GDK101** sensor platform allows you to use your GDKA101 sensor
+(`AliExpress`_, `Amazon`_, `DesertCart`_, `RadonShop`_, `FTLab`_) to monitor radiation levels at your place.
+
+The GDK101 can measure average dose rate per 1 and 10 minutes, it also detect vibrations that can disturb the measurement.
+
+Component/Hub
+-------------
+
+The GDK101 using I²C protocol for data communication.
+Connect your sensor module to MCU using I²C pins and setup your esphome to use apropariet pins for I²C communication.
+Set the I²C adress on module using jumpers and also set it up in esphome configuration.
+
+Module Pins
+-----------
+
+============  ===============================================================
+ Module Pin   Description
+============  ===============================================================
+TX            UART TX pin
+RX            UART RX pin
++5VIN         Positive supply voltage
+GND1          Ground
+SDA           I²C data bus
+SCL           I²C clock bus
+
+A0            Adress Jumper 1
+A1            Adress Jumper 2
+============  ===============================================================
+
+============  ===============================================================
+I2C Address   Jumpers State
+============  ===============================================================
+0x18          A0 Short, A1 Short
+0x19          A0 Open,  A1 Short
+0x1A          A0 Short, A1 Open
+0x1B          A0 Open,  A1 Open
+============  ===============================================================
+
+.. figure:: images/gdk101.jpg
+    :align: center
+    :width: 50.0%
+
+    GDK101 Gamma Radiation Sensor Module.
+
+.. _AliExpress: https://pl.aliexpress.com/i/1005001299979570.html
+.. _Amazon: https://www.amazon.com/GDK101-Radiation-Sensor-Module-Aduino/product-reviews/B01I4RZWA8
+.. _DesertCart: https://www.desertcart.in/products/98880150-gdk-101-gamma-radiation-sensor-module-aduino-sensor-module
+.. _RadonShop: https://www.radonshop.com/ftlab-gdk101-gamma-radiation-sensor-module-arduino
+.. _FTLab: http://allsmartlab.com/eng/294-2/
+
+Sample Configuration:
+---------------------
+
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    i2c:
+      sda: GPIO21
+      scl: GPIO22
+      scan: True
+      id: bus_a
+      frequency: 100kHz
```

Comment:
We dont add dependent components in the example as users need to set them up themselves and read that documentation.
```suggestion
```